src/lib/hooks/useOnlineStatus.ts

```diff
@@ -6,7 +6,12 @@ import { uuidv4 } from '../../utils/uuid';
 import { LoggerInterface } from '../Logger';
 
 function useOnlineStatus(sdk: SendbirdChat, logger: LoggerInterface) {
-  const [isOnline, setIsOnline] = useState(window?.navigator?.onLine ?? true);
+  const [isOnline, setIsOnline] = useState(
+    // window is undefined in SSR env
+    window != null
```

Comment:
It seems like an error may occur because the "window" is not defined.
We might need to use `typeof window !== 'undefined'` to prevent this error.